.vuepress/components/HomeFeatures.vue

```diff
@@ -110,14 +110,38 @@
                 </div>
             </div>
 
+            <h2 id="cheminformatics">Cheminformatics</h2>
+
+            <a href="https://www.biorxiv.org/content/10.1101/2021.03.17.435706v1">
+                <div class="h-feature">
+                    <div class="h-feature-content">
+                        <h3>
+                            Fast and accurate genome-wide predictions and structural modeling of protein-protein
+                            interactions using Galaxy
+                            <sup><font color="red">new</font></sup>
+                        </h3>
+                        <p>
+                            Identifying interacting proteins reveals insight into living organisms and yields novel drug
+                            targets for disease treatment. This is a publicly available, automated pipeline to predict
```

Comment:
@guerler I just took this snippet from the paper -- you may want to massage it here.